src/interfaces.ts

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import Tester from './tester';
+import ConfigurationClass from './ConfigurationClass';
+
+export type ComponentClass = React.FC | { new(): Component<any> };
```

Comment:
Would `React.ReactNode` again be a good type ?
I'm not sure if that's going to handle all the situations, but let's give it a shot !

If you're not sure let's leave it as is ...

Reply:
Looks like it! Thanks

Reply:
Actually I checked and no. :/ It doesn't like it